cpp/include/raft/linalg/qr.cuh

```diff
@@ -70,6 +69,51 @@ void qrGetQR(const raft::handle_t& handle,
 {
   detail::qrGetQR(handle, M, Q, R, n_rows, n_cols, stream);
 }
+
+/**
+ * @brief compute QR decomp and return only Q matrix
```

Comment:
Can we spell out the decumentation more clearly? I'd prefer for clarity over potential redundancy. Something like `Compute the QR decomposition of matrix M and return only the Q matrix.`